### ISSUE TITLE
网络结构问题

### ISSUE THREAD

**Issue by Dummy-design** [Guo-Stone/MambaMorph#6]
![{23417446-8E87-45cf-9694-8F7E419F8BC4}](https://github.com/Guo-Stone/MambaMorph/assets/109321679/aa6aa5e4-c771-4843-8fb7-1f3f443011a0)

Thank you for your contribution to this work. I have a question: by incorporating the DICE score between labels as part of the loss function, does this imply that the network is a semi-supervised registration network rather than a traditional unsupervised registration network? I would be grateful for your response.

**Comment by Guo-Stone** [comment]
Yeah, the training approach is NOT unsupervised. To be accurate, MambaMorph is trained in a weak-supervised manner instead of semi-supervised. The definition of weak-supervision is that we don't supervise the model's output (i.e. deformation field in this context) directly, but supervise the label with dice loss.

**Comment by Dummy-design** [comment]
Thank you for your reply. I noticed that the loss fluctuates significantly while reproducing your work. Is this a normal phenomenon?
![{57FF54E3-6D8A-4c35-91DD-B474D2D65728}](https://github.com/Guo-Stone/MambaMorph/assets/109321679/70efe9df-f32a-415e-b197-ff2d72611b83)

**Comment by Guo-Stone** [comment]
If I were you, I would plot the loss of several epochs. Only that I can see the trend clearly.
